test/javascripts/test_helper.js

```diff
@@ -105,6 +105,8 @@ function resetSite(siteSettings, extras) {
   Site.resetCurrent(Site.create(siteAttrs));
 }
 
+let baseTransalations = I18n.translations[I18n.locale].js || {};
```

Comment:
This isn't copying them right? It's a reference to that object. So if that object is mutated setting it back does nothing?

Reply:
Oh yes I changes the impelmentation of this at the end and didnt check but yes you are right I think. This one doesn’t matter that much anyways, it mosty prevents to carry locales over time, will come up with something better.